modules/distr-tracing-tempo-remove-web-console.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies:
+//
+//* distr_tracing_install/dist-tracing-tempo-removing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-removing-tempo-instance_{context}"]
+= Removing a TempoStack instance by using the OpenShift web console
+
+You can remove a TempoStack instance in the *Administrator* view of the OpenShift web console.
+
+.Prerequisites
+
+* You are logged in to the OpenShift web console as a cluster administrator with the `cluster-admin` role.
+
+.Procedure
+
+. Go to *Operators* -> *Installed Operators* -> *{TempoOperator}* -> *TempoStack*.
++
+[TIP]
+====
+You can use the *Project:* drop-down list to select the relevant OpenShift project.
+====
+
+. Remove the TempoStack instance: go to {kebab} -> *Delete TempoStack* -> *Delete*.
```

Comment:
```suggestion
. To remove the TempoStack instance, select {kebab} -> *Delete TempoStack* -> *Delete*.
```

Reply:
Thank you. This suggestion has been outdated due to my latest commit, so I'm applying it manually.